src/@ionic-native/plugins/jins-meme/index.ts

```diff
@@ -0,0 +1,169 @@
+import { Cordova, Plugin } from '@ionic-native/core';
+import { Observable } from 'rxjs/Observable';
+
+declare var cordova: any;
+
+/**
+ * @name JinsMeme
+ * @description
+ * Implementation of the JINS MEME SDK
+ *
+ * @usage
+ * ```
+ * import { JinsMeme } from 'ionic-native';
+ *
+ * JinsMeme.functionName('Hello', 123)
```

Comment:
Please replace this usage block with actual usage example.

Also, I notice you're using the old template from Ionic Native 2.x.

Usage should look like:
```typescript
import { JinsMeme } from '@ionic-native/jins-meme';

constructor(private jinsMeme: JinsMeme) { }

...

this.jinsMeme.functionName()
  .then(...)
  .catch(...);
```

See how other plugins are documented.